sm/sm.go

```diff
@@ -45,7 +46,9 @@ func NewServer(ctx context.Context, serverEnv server.Environment) (*server.Serve
 	if err != nil {
 		return nil, fmt.Errorf("Error using storage: %v", err)
 	}
+
 	defaultAPI := api.Default(storage)
+	defaultAPI.RegisterControllers(info.NewController(serverEnv))
```

Comment:
Shouldn't this be returned by *api.Default*?

Reply:
I didn't want to overload the api with both a storage and an environment. But yes, you're correct that this is a core part of the API and should be in the default. No going around passing the environment for now